04-Addressing control failure/README.md

```diff
@@ -1,4 +1,25 @@
 > The Azure Tenant Security Solution (AzTS) was created by the Core Services Engineering & Operations (CSEO) division at Microsoft, to help accelerate Microsoft IT's adoption of Azure. We have shared AzTS and its documentation with the community to provide guidance for rapidly scanning, deploying and operationalizing cloud resources, across the different stages of DevOps, while maintaining controls on security and governance.
 <br>AzTS is not an official Microsoft product – rather an attempt to share Microsoft CSEO's best practices with the community.
+<br>
+# Addressing control failure
 
-Coming soon..!!
\ No newline at end of file
+## On this page:
+
+- [Overview](README.md#overview)
+
+## Overview
+
+The AzTS solution provides a UI-based tool that can be used by dev-engineers to perform on-demand scans to verify fixes sooner, check reasons for control failures and view the latest scan results. This tool leverages your current subscription permissions to show you subscriptions that you have the ability to request scans for. 
+
+To check reason about control failure, you can refer Status Reason column in AzTS UI.
```

Comment:
If we can highlight status reason in example, that would be good. Because our masking looks like a box.